src/Glpi/Form/Migration/FormMigration.php

```diff
@@ -534,13 +535,14 @@ private function processMigrationOfComments(): void
     }
 
     /**
-     * Update horizontal rank of questions and comments to be consistent with the new form system
+     * Sets horizontal_rank to null for blocks that are contained in a horizontal layout
+     * and have only one element.
      *
      * @return void
      */
-    private function updateBlockHorizontalRank(): void
+    private function removeUselessHorizontalRanks(): void
     {
-        $this->progress_indicator?->setProgressBarMessage(__('Updating horizontal rank...'));
+        $this->progress_indicator?->setProgressBarMessage(__('Removing useless horizontal ranks...'));
```

Comment:
Maybe this message is a bit too technical, I would prefer something generic like "Normalizing layout" (open to better suggestions).

Reply:
`Cleaning imported data...`.